app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,13 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.new(params.require(:comment).permit(:text))
```

Comment:
- [ ] [OPTIONAL] Here you can apply the same I mentioned for the posts.

Reply:
Done :+1: Thank you :smile: